RDKShell/RDKShell.cpp

```diff
@@ -3421,7 +3421,7 @@ namespace WPEFramework {
 #endif
                 }
 
-                if (!type.empty() && type == "SearchAndDiscoveryApp" )
```

Comment:
Containerisation of JSPP should be done separately if needed. Please remove this change